docs/source/using/profiles.adoc

```diff
@@ -12,7 +12,26 @@ toc::[]
 == Overview
 
 {project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+Each {project} instance can be created with different configurations e.g memory, CPU, disk-size , add-ons etc and users can switch between different instances.
+Profile feature would help users to test different configurations without deleting an existing {project} instance.
+
+Each profile is completely independent of other profiles and things like cache, add-ons and persistent config are specific to only one profile.
+This means if you have set some specific configuration parameters for a profile, these settings do not apply to other profiles.
+
+User can have mutiple profiles but only one profile will be active at one point of time i.e. by default {project} commands will run against the active profile.
```

Comment:
"You can ..." - we try as much as possible to directly address the user. 
I am missing a bit of a red thread in this and the following sentences. IMO they are not connected so well

Reply:
I think we are not consistent in the use of "instance" vs "profile". For example Create Profile section. Since we defined what a profile is and the section is called "creating profile", it should really be "you can create a new profile. ..."